Marlin/src/feature/pause.cpp

```diff
@@ -143,16 +143,18 @@ bool pause_print(const float &retract, const float &z_lift, const float &x_pos,
   // Save current position
   stepper.synchronize();
   COPY(resume_position, current_position);
-
+  set_current_to_destination();            // Reset all old destinations to current
```

Comment:
Here we initialise destinations and current to avoid the confusion to use the previous destination used by the previous process